.github/workflows/changelog-check.yaml

```diff
@@ -0,0 +1,33 @@
+name: CHANGELOG Checks
+on:
+  pull_request:
+    paths:
+      - CHANGELOG.md
+
+jobs:
+  changelog-check:
+    name: CHANGELOG Check
+    runs-on: ubuntu-latest
+    steps:
+      # - name: Comment PR
```

Comment:
Do we need this whole commented section? Maybe just remove it?

Reply:
It was for tests. I couldn't make it work. So I removed it for now. I'll try it later.